packages/block-library/src/latest-posts/editor.scss

```diff
@@ -4,6 +4,15 @@
 		padding-left: 0;
 	}
 }
-.wp-block-latest-posts li a > div {
-	display: inline;
+
+.wp-block-latest-posts {
+
+	li a > div {
+		display: inline;
+	}
+
+	.wp-block-latest-posts__post-excerpt {
```

Comment:
I wonder if we can make this a little less specific.

Right now if you want to override the margins of this excerpt, you have to write:

`.wp-block-latest-posts .wp-block_latest-posts__post-excerpt`

Could we do with just the latter half?

Reply:
Good call. Updated in 7a38608